icu4c/source/i18n/unicode/measunit.h

```diff
@@ -52,13 +249,41 @@ class U_I18N_API MeasureUnit: public UObject {
      * @stable ICU 3.0
      */
     MeasureUnit(const MeasureUnit &other);
-        
+    
     /**
-     * Assignment operator.
+     * Move constructor.
+     * @stable ICU 3.0
```

Comment:
should be draft 67, and inside hide-draft, right? or did we get a default impl?

Reply:
done